sans2dVacTankApp/Db/sans2d_vac_stop_detectors_baffles.template

```diff
@@ -6,17 +6,32 @@ record(calcout, "$(FULL_PREFIX)STOP_$(MOVING_SHORT)$(STATIONARY_SHORT):_CALC"){
     field(INPA, "$(FULL_PREFIX)$(INTERVAL_NAME):INTERVAL CP")
     field(INPB, "$(FULL_PREFIX)$(INTERVAL_NAME):MINIMUM_INTERVAL CP")
     field(INPC, "$(P)$(MOVING_AXIS):MTR.TDIR CP")
-    field(INPD, "$(FULL_PREFIX)COLLISION_AVOIDANCE")
+    field(INPD, "$(FULL_PREFIX)COLLISION_AVOIDANCE CP")
 
     field(CALC, "((A <= B) && (C == $(COLLISION_DIRECTION)) && (D == 0)) ? 1 : 0")
 
     field(OOPT, "When Non-zero")
-    field(OUT, "$(FULL_PREFIX)STOP_$(MOVING_SHORT)$(STATIONARY_SHORT):_FAN PP")
+    field(OUT, "$(FULL_PREFIX)STOP_$(MOVING_SHORT)$(STATIONARY_SHORT):_SEQ.PROC PP")
 }
 
-record(dfanout, "$(FULL_PREFIX)STOP_$(MOVING_SHORT)$(STATIONARY_SHORT):_FAN") {
-    field(OUTA, "$(P)$(MOVING_AXIS):MTR.STOP PP")
-    field(OUTB, "$(FULL_PREFIX)_LAST_STOP_TIME.PROC PP")
+# Record to store old acceleration while we send stop
+record(ai, "$(FULL_PREFIX)STOP_$(MOVING_SHORT)$(STATIONARY_SHORT):_OLDACEL") {}
+
+record(sseq, "$(FULL_PREFIX)STOP_$(MOVING_SHORT)$(STATIONARY_SHORT):_SEQ") {
```

Comment:
Why are the waits commented out?
If we're sending a stop here as well as in the hardware will this stop not use the old deceleration? You then have the situation where you're sending a fast stop while your slow stop is still ongoing, I'm not sure what the behaviour is here but probably needs further testing.